### DISCUSSION TITLE
Having a problem running VCL demo

### DISCUSSION THREAD

**Discussion by ZaakaJarimbo** [galfar/imaginglib#33] [Q&A]
I have just downloaded and setup the Imaging library, however, when trying to run the VCL Demo I get errors :( see attached link, does anyone know what I have done wrong?

[https://www.teeseyed.online/i1.jpg](url)
![Imaging1](https://user-images.githubusercontent.com/5955746/212446965-1b7460a6-5ee0-4d6a-853d-0a7d82d776e8.jpg)

**Comment by galfar** [comment]
Sorry for the very late reply, I don't seem to get notifications from GitHub discussions.

I could not reproduce this with Delphi 11.3, which version exactly did you use?

**Comment by DeerBear** [comment]
Hey! 

_That code **SMELLS** badly_, _**even if**_ (and _I have doubts_, but I also don't have it) **it works in 13**. 
The fundamental issue is that you're calling _**a class method**_ (Create) in **_an instance method_** (assign) and _even if it worked that's a huge **NONO_**.

I'm not sure what the intent was, but I promise you that ain't the solution.

Cheers!
